### DISCUSSION TITLE
Not Able to See plausible-conf thing and docker-compose thing.

### DISCUSSION THREAD

**Discussion by honey7988023** [plausible/analytics#4677] [Self-Hosted Support]
Hi Everyone, My Issue is that whenever docker-compose.yml and plausible-conf.env i see nothing. I do not know what to do next.  I am not seeing base url, secret base url and other kind of things when open with editor like nano

**Comment by honey7988023** [comment]
@ruslandoga can you please help me brother

**Comment by ruslandoga** [comment]
👋 @honey7988023 

The example Docker Compose configuration no longer uses `plausible-conf.env`

Please see https://github.com/plausible/community-edition for the up-to-date guide.

**Comment by honey7988023** [comment]
I Did not get how to self host plausible now. Can you share me link or
video where they teach how to self host plausible in a new way

**Reply by ruslandoga** [reply]
No, sorry. You can copy paste the linked guide to ChatGPT and it would answer any questions you have.

**Comment by honey7988023** [comment]
I pasted on chatgpt and it shows nothing. I can pay you can you please
install self host plausible for me or do you know someone who can do it for
me.
